apis/v1beta1/validation/httproute.go

```diff
@@ -40,6 +40,9 @@ var (
 
 	// All valid path characters per RFC-3986
 	validPathCharacters = "^(?:[A-Za-z0-9\\/\\-._~!$&'()*+,;=:@]|[%][0-9a-fA-F]{2})+$"
+
+	// Minimum value of timeout in milliseconds
+	TimeoutMinValue = 1
```

Comment:
Should this be lower case?
```suggestion
	timeoutMinValue = 1
```

Reply:
addressed in 05f32daa4ee6b188d6f8d07c3777d8ececdb786d